test/Main.purs

```diff
@@ -4,16 +4,14 @@ import Prelude
 import Effect (Effect)
 import Effect.Console (log)
 import Data.Newtype (class Newtype, unwrap)
-import Type.Equality (class TypeEquals, proof, to, from)
+import Type.Equality (class TypeEquals, proof)
 
 newtype RecordNewtype = RecordNewtype
   { message :: String }
 
 instance newtypeRecordNewtype ::
-  TypeEquals inner { message :: String }
-    => Newtype RecordNewtype inner where
-  wrap = RecordNewtype <<< to
-  unwrap (RecordNewtype rec) = from rec
+  (TypeEquals inner { message :: String }
+  ) => Newtype RecordNewtype inner
```

Comment:
There’s no need for those parens now that the Coercible constraint is introduced by TypeEquals itself.

Reply:
Done.